en/streaming-integrator/mkdocs.yml

```diff
@@ -234,7 +235,15 @@ nav:
     - 'Deploying Siddhi Applications': 'develop/deploying-Streaming-Applications.md'
     - 'Exporting Siddhi Applications': 'develop/exporting-Siddhi-Applications.md'
   - 'Observability':
-    - 'Monitoring the Streaming Integrator': 'admin/monitoring-the-streaming-integrator.md'
+    - 'Monitoring SI via Grafana Dashboards':
+      - 'Configuring Grafana Dashboards': 'admin/setting-up-grafana-dashboards.md'
+      - 'Managing Grafana Dashboards': 'admin/managing-grafana-dashboards.md'
+      - 'Viewing Grafana Dashboards':
+        - 'Viewing Overview Statistics': 'admin/viewing-overview-statistics.md'
+        - 'Viewing CDC Statistics': 'admin/viewing-cdc-statistice.md'
+        - 'Viewing File Statistics': 'admin/viewing-file-statistice.md'
+        - 'Viewing RDBMS Statistics': 'admin/viewing-rdbms-statistice.md'
+    - 'Monitoring SI via Analytics Dashboard: 'admin/monitoring-the-streaming-integrator.md'
```

Comment:
There is one apostrophe missing in 'Monitoring SI via Analytics Dashboard'.
This might break the build...

Reply:
Added the missing apostrophe.